brian_cheung/week_two_challenges/more_methods.md

```diff
@@ -64,11 +93,24 @@ big_ordered_array(test_array)
 # output => ['I', 'OF', 'FULL', 'LOVE', 'ARRAYS', 'MAKING', 'STRINGS']
 ```
 ```ruby
-# write your code here
+def big_ordered_array (array)
+   puts array.map {|word| word.upcase }.sort.sort_by { |word| word.length }
+end
 ```
 
+#old code for refrence
+#
+#def big_ordered_array (array)
+#  puts array.map {|word| word.upcase }.sort_by { |word| word.length }.group_by(&:length).map{ |iterate, array| #array.sort}.flatten
+#end
+
 4. Declare a method called `lucky_number` that takes two numbers as arguments and returns "Unlucky!" if the sum of those numbers is divisible by 13, and "Lucky!" if not.
 
 ```ruby
-# write your code here
+def lucky_number (number1, number2)
+  if (number1 + number2) % 13 == 0
+    puts "Unlucky!"
+  else
+    puts "Lucky!"
+end
```

Comment:
One last thing: you're missing something here that will cause this code to break...

Reply:
...crap.. whoops...